stats/statsclient/src/main/java/ru/practicum/client/StatsClient.java

```diff
@@ -0,0 +1,73 @@
+package ru.practicum.client;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import ru.practicum.dto.StatParam;
+import ru.practicum.dto.StatsDto;
+import ru.practicum.dto.StatsOutDto;
+
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class StatsClient implements BaseClient {
+    private final String serverUrl = "http://localhost:9090";
+
+    private final String dateTimeFormat = "yyyy-MM-dd HH:mm:ss";
```

Comment:
FYI
Константы в java обычно имеют сочетание модификаторов [static final](https://www.theserverside.com/video/Why-we-use-static-final-in-Java-for-constants#:~:text=The%20static%20keyword%20means%20the,to%20create%20a%20constant%20value.) и именуются в CAMEL_CASE
```suggestion
    private static final String DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm:ss";
```